streampipes-extensions-management/src/main/java/org/apache/streampipes/extensions/management/connect/adapter/model/pipeline/AdapterEventPreviewPipeline.java

```diff
@@ -79,4 +80,9 @@ public Map<String, GuessTypeInfo> makePreview() {
         .collect(Collectors.toMap(Map.Entry::getKey,
             e -> new GuessTypeInfo(e.getValue().getClass().getCanonicalName(), e.getValue())));
   }
+
+  @Override
+  public EventSchema getResultingEventSchema() {
+    return null;
```

Comment:
Why does this method always return null?

Reply:
This is only temporary. In the future we probably will replace the class `AdapterEventPreviewPipeline` with a regular `AdapterPipeline` that uses a different sink.
I had to add it here to remove the dependency of the `FileStreamProtocol` with the internal adapter implementation.
If you have an idea on how to remove it, I'm happy to discuss it. If there is no quick solution, I would suggest to leave it as it is and remove it during the revision of the connect API.